README.md

```diff
@@ -52,6 +52,7 @@ Access design documentation for features slotted in the 4.0 release of OpenShift
 - [Environment](http://openshift.github.io/openshift-origin-design/web-console/4.0-designs/environment/environment)
 - [Forms](http://openshift.github.io/openshift-origin-design/web-console/4.0-designs/forms/forms)
 - [Global Configuration](http://openshift.github.io/openshift-origin-design/web-console/4.0-designs/global-config/global-config)
+- [Guset Agent Not Installed](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/guest-agent-not-installed/guest-agent-not-installed)
```

Comment:
can you move this to the **KubeVirt (Virtualization in OpenShift)** section.

Reply:
And a nit: “Guset” should be “Guest”

Reply:
Done. Thanks, @matthewcarleton @andybraren

Reply:
@yfrimanm do you mind updating the design to have the text say "Although this virtual machine is running the IP address is not available because the guest agent isn't installed" 

The way it is worded now makes it sound like the VM isn't available at all when I think it technically is.

Reply:
IIRC the Windows guest agent is also necessary to enable certain management features like graceful restarts, and probably more.

I mocked up some guest agent popover text [here](https://github.com/openshift/openshift-origin-design/blob/master/web-console/4.0-designs/status/status.md) but we can change it of course.

> This virtual machine does not have a guest agent installed within its guest operating system. This will restrict some management capabilities.

Reply:
Two others nits:

1. The status icons of the other two “Running” VMs use the old lightning bolt icon.

2. “Learn more” should have an external link icon next to it, since it links outside of the console. Example at the bottom [here](https://github.com/openshift/openshift-origin-design/blob/master/web-console/4.0-designs/status/status.md).

Should these changes be made in a separate PR to make this one easier to merge?

Reply:
+1 I like that text. The only downfall to not mentioning IP specifically is then it could be confusing that it's not available. Which might mean we bring back the tooltip, which I'm not crazy about. WDYT @andybraren ?
ya I think we can open up issues for these changes. 
We've already included the icon update in [this issue](https://github.com/openshift/openshift-origin-design/issues/163)

Reply:
I'm also not crazy about tooltips. 😄

[Here's some other information](https://access.redhat.com/solutions/261763) that isn't available within RHV when a guest agent is missing. Memory usage, CPU utilization, Spice console networking auto-detection, etc.

Here's a simpler message that would include the above:

> This virtual machine does not have a guest agent installed. Some usage metrics, networking information, and management features will not be available.

I'm tempted to say that the blue `Running*` status within list views, PF3 alert at the top of the current VM Details page, and Health card of the eventual VM dashboard would at least ensure that the user doesn't stay confused about the IP being `Unavailable` for very long. I admit it's a problem, but maybe those two other avenues for discovery are okay? I'm also not sure tooltips on `Unavailable` cells is an existing pattern.

Should we add this as a usability test item? If testing shows we need a tooltip, then I'd concede.

Reply:
how do you guys have so many comments on a single line :D Am i missing something?

Reply:
> 1. The status icons of the other two “Running” VMs use the old lightning bolt icon.

Sorry, @andybraren, I don't understand what you mean: I doubled checked the icons again both in our PF4 Sketch library and also in the PF website and the icon that is shown for "running" is the same one I used. 
![image](https://user-images.githubusercontent.com/44577420/58469017-5af24a00-8147-11e9-8777-b510fdea0f8b.png)
If you are referring to another one, would you please send me a link to it? 
Thank you.

Reply:
@serenamarie125 that was my bad 🤦‍♂